client/webserver/site/src/js/locales.ts

```diff
@@ -111,6 +112,7 @@ export const enUS: Locale = {
   [ID_BUY]: 'Buy',
   [ID_SELL]: 'Sell',
   [ID_NOT_SUPPORTED]: '{{ asset }} is not supported',
+  [ID_VERSION_NOT_SUPPORTED]: 'version: {{ version }} of {{ asset }} is not supported',
```

Comment:
Let's rephrase this.  The "version: " prefix there is a bit odd.  How about:

> {{ asset }} version {{ version }} is not supported

e.g. "USDC.ETH version 0 is not supported"

or

> {{ asset }} (v{{version}}) is not supported

e.g. "USDC.ETH (v0) is not supported"

Any preferences?

I think the point of the issue this PR is aiming to resolve is that without naming the version it might be confusing if the user can see that the server does indeed have a market for this asset, so we have to state the version so there's some kind of explanation for why it's not supported.